sky/clouds/service_catalog/images/aws_image_gen.py

```diff
@@ -0,0 +1,82 @@
+"""
+Run `python aws_image_gen.py` to copy the AMI in the source region to all regions and make them public.
+Will also generate a `images.csv` or append to an existing one; then we can refresh the `images.csv` inSkyPilot Catalog.
+"""
+
+import csv
+import json
+import os
+import subprocess
+import time
+
+# Inputs based on Packer generated AMI
+IMAGE_REGION = 'us-east-1'
+IMAGE_ID = 'ami-0981cc842c7188227'
+BASE_IMAGE_ID = 'ami-005fc0f236362e99f'
+OS_TYPE = 'ubuntu'
+OS_VERSION = '22.04'
+
+OUTPUT_CSV = 'images.csv'
+ALL_REGIONS = [
+    'us-west-1',
+]
```

Comment:
How about making these values, especially the `BASE_IMAGE_ID` an arg of this script, so that in our instruction, we can directly pass them into the script in command line, instead of asking people to update the variable in the python script.